### ISSUE TITLE
Problems adding a new user

### ISSUE THREAD

**Issue by gilesbradshaw** [RocketChat/Rocket.Chat#17136]
### Description:

If you add a new user with set random password and email selected it doesn't  - they just get the welcome email

(would be good for this to tell them what their account id is btw)

### Steps to reproduce:

As an admin add a new user - leave the set random password and email selected

1. 
2. 
3. 

### Expected behavior:

The user should be emailed a random password.

### Actual behavior:
They just get the standard email.

If you then edit the password (as admin) and they try and log in with new one that doesn't work either.

Basically adding users as admin doesn't seem to work...

### Server Setup Information:

- Version of Rocket.Chat Server: rocketchat/rocket.chat:latest (just downloaded)
- Operating System: ubuntu
- Deployment Method:  docker
- Number of Running Instances: §
- DB Replicaset Oplog: recketchat rs0
- NodeJS Version: whatever is in your image
- MongoDB Version: mongo:4.0

### Client Setup Information

- Desktop App or Browser Version: Browser (chrome)
- Operating System: ubuntu

**Comment by jschirrmacher** [comment]
I can confirm that behavior with version 3.1.1
